plugin/pkg/auth/authorizer/rbac/bootstrappolicy/testdata/controller-roles.yaml

```diff
@@ -872,6 +872,7 @@ items:
     resources:
     - nodes
     verbs:
+    - get
```

Comment:
Auth changes should go to a separate commit.

Reply:
Done